django/utils/crypto.py

```diff
@@ -17,7 +18,11 @@
 
 # Use the system PRNG if possible
 try:
-    random = random.SystemRandom()
+    if sys.version_info >= (3, 6):
+        import secrets
+        random = secrets.SystemRandom()
+    else:
+        random = random.SystemRandom()
```

Comment:
I'm in favor of adding this if the benefits become more than theoretical or when only Python 3.6+ is supported.